deepchem/models/layers.py

```diff
@@ -2954,18 +2968,10 @@ def build(self, input_shape):
       else:
         self.dropouts.append(None)
       prev_layer_size = layer_size
-    self.W_list.append(
```

Comment:
I think this might be a merge error. I think this is undoing the fix to the DAG model from the previous PR